sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/LibraryTasks.scala

```diff
@@ -78,6 +78,11 @@ object LibraryTasks {
         webpackResourceFiles ++ compileResources
       val cacheLocation = streams.value.cacheDirectory / s"${stage.key.label}-webpack-libraries"
       val extraArgs = (webpackExtraArgs in stage).value
+      val webpackMode = if ((scalaJSLinkerConfig in stage).value.semantics.productionMode) {
+        Webpack.DevelopmentMode
+      } else {
+        Webpack.ProductionMode
```

Comment:
I think the `then` and `else` part need to be switched :P

Reply:
Doh,  should be fixed now